language_it-IT.js

```diff
@@ -0,0 +1,64 @@
+    NOTHING_PLAYING: "Non c'è musica in riproduzione.",
+
+    PLAY_SUCCESS: "Ok, inizio la riproduzione.",
+    PLAY_ERROR: "Qualcosa non ha funzionato. Forse qualcosa è già in riproduzione?",
+
+    PLAYONDEVICE_SUCCESS: "Ok, sto riproducendo su {{deviceName}}",
+    PLAYONDEVICE_NO_MATCH: "Non ho trovato il dispositivo richiesto, riprova.",
+    PLAYONDEVICE_NO_MATCH_REPROMPT: "Riprova. Dimmi il nome del dispositivo su cui vuoi iniziare la riproduzione.",
+
+    PAUSE_SUCCESS: "Ok, metto in pausa.",
+    PAUSE_ERROR: "Qualcosa non ha funzionato. Forse la musica è già in pausa?",
+
+    NEXTSONG_SUCCESS: "Ok, successivo.",
+
+    PREVIOUSSONG_SUCCESS: "Ok, precendente.",
+
+    SETVOLUME_SUCCESS: "Ok, imposto il volume al {{volume}}%.",
+
+    TOGGLESHUFFLE_ON: "Ok, riproduzione casuale attiva.",
+    TOGGLESHUFFLE_OFF: "Ok, riproduzione casuale disattiva.",
+
+    SEEK_SUCCESS: "Okay.",
+
+    ERR_NO_TOKEN: "Non hai ancora collegato questa skill al tuo account Spotify. Per farlo, apri l'app Alexa e segui le istruzioni.",
+    ERR_UNKNOWN: "Errore, riprova più tardi.",
+
+    // https://developer.spotify.com/documentation/web-api/reference/object-model/#player-error-object
+    ERR_REASON: {
+      NO_PREV_TRACK: "Questo comando richiede una traccia precendente, ma non ce n'è nessuna nella playlist.",
+      NO_NEXT_TRACK: "uesto comando richiede una traccia successiva, ma non ce n'è nessuna disponibile nella playlist.",
+      // NO_SPECIFIC_TRACK: "La traccia richiesta non esiste.",
+      ALREADY_PAUSED: "Questo comando richiede che la musica non sia già in pausa.",
+      NOT_PAUSED: "Questo comando richiede che la musica sia in pausa.",
+      // NOT_PLAYING_LOCALLY: "Questo comando richiede della musica in riproduzione su questo dispositivo.",
+      NOT_PLAYING_TRACK: "Questo comando richiede una traccia attualmente in riproduzione.",
+      NOT_PLAYING_CONTEXT: "Questo comando richiede che una playlist sia attualmente in riproduzione.",
+      ENDLESS_CONTEXT: "La riproduzione casuale non può essere attivata su una playlist senza fine.",
+      CONTEXT_DISALLOW: "Non è stato possibile eseguire questo comando.",
+      // ALREADY_PLAYING: "La traccia non dovrebbe essere ricominciata da capo se la stessa traccia e la stessa playlist sono già in riproduzione e se c'è un punto di ripresa.",
+      // RATE_LIMITED: "Sei al momento limitato per un numero eccessivo di richieste.",
+      REMOTE_CONTROL_DISALLOW: "La riproduzione non può essere controllata.",
+      DEVICE_NOT_CONTROLLABLE: "Non è possibile controllare questo dispositivo da remoto.",
+      VOLUME_CONTROL_DISALLOW: "Non è possibile controllare il volume di questo dispositivo.",
+      NO_ACTIVE_DEVICE: "Non c'è nessun dispositivo disponibile su cui eseguire questo comando.",
+      PREMIUM_REQUIRED: "Sfortunatamente questa richiesta è disponibile solo per gli utenti premium.",
```

Comment:
Please somehow mention that it's about _Spotify premium_, not e.g. a premium version of this skill. I missed that while translating from German to English.
```suggestion
      PREMIUM_REQUIRED: "Sfortunatamente questa richiesta è disponibile solo per gli utenti Spotify premium.",
```

Reply:
Unresolving since this isn't fixed yet.

Reply:
Oh sorry, I thought you did fix it